test/integration/test-amp-ad-3p.js

```diff
@@ -24,7 +24,8 @@ function createFixture() {
   return createFixtureIframe('test/fixtures/3p-ad.html', 3000, () => {});
 }
 
-describe('amp-ad 3P', () => {
+//TODO(#30528): unskip this for IE when possible
```

Comment:
Does this fail after transforms? Or is IE non-compatible with the transform?

Reply:
hmm i don't remember, i'll have to rerun it again

Reply:
so i removed the change and its green now so it might have been a flake or something i fixed with the subsequent changes.